docs/source/author/topics.rst

```diff
@@ -80,1007 +76,634 @@ the project's topic application, where:
-      must know the content covered in the listed lessons.
-      Since curriculum integrations are stored at a topic level, both the unit
-      plan slug and lesson slug must be provided.
-      The slugs should be separated by the ``/`` character, see example
-      below.
-      The slugs are defined in the :ref:`unit-plan-file`.
+    - **Required Fields:**
 
-      - For example:
+      - ``name:`` The name of the curriculum area (this is what will be displayed to the
+        user).
 
-        .. code-block:: yaml
+    - **Optional Fields:**
 
-          prerequisite-lessons:
-            - unit-plan/introduction-to-bits
-            - unit-plan/counting-bits
+      - ``children:`` A list of sub-curriculm areas (see example file below). Each child
+        requires a name field.
 
-An example curriculum integrations configuaration file with multiple activities
-may look like the following:
+An example curriculum areas file with multiple curriculums may look like
+the following:
 
 .. code-block:: yaml
 
-  binary-number-bracelets:
-    number: 1
-    md-file: bracelets.md
-    curriculum-areas:
-      - arts
-      - design
-    prerequisite-lessons:
-      - unit-plan/introduction-to-bits
-      - unit-plan/counting-bits
-
-  hidden-binary-signals:
-    number: 2
-    md-file: hidden-binary-signals.md
-    curriculum-areas:
-      - science
-
-.. _programming-exercises-structure-file:
-
-Programming Exercises Structure File
-------------------------------------------------------------------------------
-
-This file defines the structure of programming exercises for all topics.
-The two components it defines is available language for exercise
-implementations, and difficulties of exercises.
+  maths:
+    name: Maths
+    children:
+      geometry:
+        name: Geometry
+      algebra:
+        name: Algebra
 
-The location of this file is within the language directory.
-This configuaration file is listed in a :ref:`application-structure-file`.
-This file can contain as many languages and difficulties as you like.
+  science:
+    name: Science
 
-The file should have the following key/value pair structure:
+  art:
+    name: Art
 
-- ``languages`` - A key containing all language slugs
-  (see :ref:`what-is-a-slug`) for available languages for implementations of
-  programming execises.
-  These slugs contain the language data.
+.. note::
 
-  - For example:
+  The maximum depth for children is one, that is, children curriculum areas
+  cannot have children.
 
-    .. code-block:: yaml
+.. note::
 
-      lessons:
-        python:
-          ...language data here...
-        scratch:
-          ...language data here...
+  When including a curriculum area in another configuration file, adding a child
+  curriculum area will automatically add the parent curriculum area, you do not need to
+  specify this manually. For example, adding "geometry" means that "maths" is
```

Comment:
Would it be better to have `geometry` and `maths` in code blocks, as we are referencing keys?